hack/run-functests.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-# All test suits should be enabled once the bug https://bugzilla.redhat.com/show_bug.cgi?id=1865839 fixed
-# GINKGO_SUITS=${GINKGO_SUITS:-functests}
-GINKGO_SUITS=${GINKGO_SUITS:-"functests/0_config functests/1_performance functests/3_performance_status"}
+GINKGO_SUITS=${GINKGO_SUITS:-functests}
+GINKGO_SUITS="functests/0_config functests/2_performance_update"
```

Comment:
why did you remove 1 and 3 in favour of 2 ?

Reply:
same question. Everything else LGTM!

Reply:
my bad added it to test locally

Reply:
done